stories/browser/src/UICellDebot/UICellDebot.tsx

```diff
@@ -69,6 +71,8 @@ const useStyles = makeStyles(() => ({
     },
     textContainer: {
         paddingTop: UILayoutConstant.contentInsetVerticalX2,
+    },
+    text: {
         alignSelf: 'stretch',
         alignItems: 'center',
```

Comment:
Shouldn't this style be moved into `textContainer` instead?

Reply:
https://github.com/tonlabs/UIKit/pull/1520/commits/92f8c96bd4f4b6cf6579f17c984fc542af55c119